### PR TITLE
Jingle test

### PR DESCRIPTION
**Pull request by JingleH**
just change this for fun@

### CODE REVIEW

**Comment by ashgarr** [comment]
this is a comment!

**Review by ashgarr — CHANGES_REQUESTED** [review]
this isn't necessary, and doesn't fit with the theming of the website

suggestions:

- add dick vitale
- yeet on 'em

(that's ok! this is a test anyway, and wouldn't be merged in the first place)

File: src/views/Verify.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Verify extends Component {
     render() {
```
Comment (ashgarr) [inline src/views/Verify.js:5]: another quick review test

File: src/views/Verify.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ class Verify extends Component {
...
         return(
             <div>
-                <p>contentless verify page</p>
+                <p>no-heatmap-jingle-test ~~~~ contentless verify page</p>
             </div>
```
Comment (ashgarr) [inline src/views/Verify.js:9]: test 2

File: src/views/Verify.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ class Verify extends Component {
     render() {
         return(
             <div>
-                <p>contentless verify page</p>
```
Comment (ashgarr) [inline src/views/Verify.js:8]: yeet

File: src/views/Verify.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ class Verify extends Component {
     render() {
         return(
             <div>
```
Comment (JingleH) [inline src/views/Verify.js:7]: haha~

**Review by ashgarr — DISMISSED** [review]
this is the same thing

**Review by ashgarr — CHANGES_REQUESTED** [review]
re-review after dismissal

**Review by ashgarr — APPROVED** [review]

**Review by ashgarr — CHANGES_REQUESTED** [review]
re-reject after test of approve button

**Pull request closed without merging**